qpylib/encdec.py

```diff
@@ -148,6 +151,9 @@ def decrypt(self):
         if 'secret' not in self.config[self.name]:
             raise ValueError("Encryption : no secret to decrypt")
 
+        if self.config[self.name].get('version') != Encryption.engine_version:
```

Comment:
I think this will fall over if `version` is not in self.config[self.name].
Also, what is a user supposed to do if they are trying to decrypt a secret that they stored using an older engine version? Is that data just lost?

Reply:
We need to support old engine versions so that old secrets can be decrypted and then re-encrypted with the latest engine. This will be covered by a separate ticket.